src/pages/index.js

```diff
@@ -17,6 +17,5 @@ const IndexPage = () => {
 
 export default IndexPage;
 
-// TODO: add gatsby-plugin-eslint
-// TODO: babel configuration
+// TODO: create screen shots from hangman and school-app
```

Comment:
maybe you can create a todo.txt file? 
todos in files could be placed in very place to be implemented.